rules/audit-policy-content/raw.rego

```diff
@@ -0,0 +1,154 @@
+package armo_builtins
+
+import future.keywords.if
+import future.keywords.in
+
+# CIS 3.2.2 https://workbench.cisecurity.org/sections/1126657/recommendations/1838583
+
+deny[msga] {
+	obj := input[_]
+	is_api_server_info(obj)
+	api_server_info := obj.data.APIServerInfo
+
+	not contains(api_server_info.cmdLine, "--audit-policy-file")
+
+	msga := {
+		"alertMessage": "audit logs are not enabled",
+		"alertScore": 5,
+		"packagename": "armo_builtins",
+		"failedPaths": [],
+		"fixPaths": [],
+		"alertObject": {"externalObjects": {
+			"apiVersion": obj.apiVersion,
+			"kind": obj.kind,
+			"metadata": obj.metadata,
+			"data": api_server_info.cmdLine,
+		}},
+	}
+}
+
+deny[msga] {
+	obj := input[_]
+	is_api_server_info(obj)
+
+	api_server_info := obj.data.APIServerInfo
+
+	contains(api_server_info.cmdLine, "--audit-policy-file")
+
+	rawPolicyFile := api_server_info.auditPolicyFile
+	policyFile = yaml.unmarshal(base64.decode(rawPolicyFile.content))
+
+	are_audit_file_rules_valid(policyFile.rules)
+
+	failed_obj := json.patch(policyFile, [{
+		"op": "add",
+		"path": "metadata",
+		"value": {"name": "Audit policy file"},
```

Comment:
Maybe it's good to add here the obj name, which is the name of the node

Reply:
I think I don't see what you mean exactly, can you give some example?

Reply:
Sure.
```rego
{"value": {"name": sprintf("%s - Audit policy file", [obj.metadata.name])}
```